packages/dashboard-backend/src/static.ts

```diff
@@ -34,7 +34,9 @@ export function registerStaticServer(publicFolder: string, server: FastifyInstan
   const doNotCache = [
     '/dashboard/',
     '/dashboard/index.html',
-    '/dashboard/client.bundle.js',
+    '/dashboard/client.js',
```

Comment:
We should remove  ```'/dashboard/client.js',```  here.

Reply:
and replace ```pathData.chunk.name === 'service-worker' || 'editor.worker'``` to ```pathData.chunk.name === 'service-worker' || pathData.chunk.name === 'editor.worker'```

https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/webpack.config.common.js#L29

Reply:
you're right, fixing it